release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md

```diff
@@ -123,4 +139,20 @@ To view bug fixes, see the GitHub milestone for 2201.2.0 (Swan Lake) of the repo
 - [update tool](https://github.com/ballerina-platform/ballerina-update-tool/issues?q=is%3Aissue+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed+label%3AType%2FBug)
 - [OpenAPI](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed)
 
+### Ballerina packages updates
+
+#### New features
+
+Added an `include` field under the `[package]` table. It allows providing paths to any additional resources, which need to be packed in the BALA file.
+
+```ballerina
+[package]
+org = "samjs"
+name = "winery"
+version = "0.1.0"
+include = [documents/”, "images/sample.png"]
+```
+
+**Info:** The `include` field accepts a string array, which contains the directory or file paths to include in the BALA. The paths should be relative to the package root directory.
```

Comment:
Shall we have this part in the description as suggested in the comment https://github.com/ballerina-platform/ballerina-release/pull/2027/files#r913165929? @praneesha WDYT?

Reply:
Removed **Info** and added that content according to the comment https://github.com/ballerina-platform/ballerina-release/pull/2027/files#r913165929.